<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDialog</class>
 <widget class="QDialog" name="ReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Modèle actif</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="canvasComboBox">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Standard</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>175</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Sauvegarder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="suprButton">
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QGraphicsView" name="reportDisplay">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>630</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Entète</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="0">
          <widget class="QPushButton" name="comButton">
           <property name="text">
            <string>Ajout Commentaire</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="nameButton">
           <property name="text">
            <string>Ajout Nom</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="dateButton">
           <property name="text">
            <string>Ajout Date</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Police</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="2" column="1" colspan="2">
             <widget class="QComboBox" name="fontComboBox">
              <item>
               <property name="text">
                <string>Arial</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Arial Black</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Book Antiqua</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Comic Sans MS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Courier New</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Georgia</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Impact</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lucida Console</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lucida Sans Unicode</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MS Sans Serif</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Palatino Linotype</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tahoma</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Times New Roman</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Trebuchet MS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Verdana</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="fontIncButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="fontDecButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="boldCheckBox">
        <property name="text">
         <string>Titres gras</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Ligne</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QPushButton" name="addLineButton">
           <property name="text">
            <string>Ajouter Ligne</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="addPicButton">
           <property name="text">
            <string>Ajouter Image</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="addTitleButton">
           <property name="text">
            <string>Ajouter Encart</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="comLineButton">
           <property name="text">
            <string>Ajouter Commentaire</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Police</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="3">
             <widget class="QPushButton" name="fontAugButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="fontRedButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QComboBox" name="fontComboBox_2">
              <item>
               <property name="text">
                <string>Arial</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Arial Black</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Book Antiqua</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Comic Sans MS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Courier New</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Georgia</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Impact</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lucida Console</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lucida Sans Unicode</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MS Sans Serif</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Palatino Linotype</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tahoma</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Times New Roman</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Trebuchet MS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Verdana</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="1">
          <widget class="QPushButton" name="printButton">
           <property name="text">
            <string>Imprimer</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pdfButton">
           <property name="text">
            <string>PDF</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>891</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>891</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>printButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>870</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>844</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
